src/hooks/coreSDK/safeCoreSDK.ts

```diff
@@ -79,6 +90,20 @@ export const initSafeSDK = async (provider: EIP1193Provider, safe: SafeInfo): Pr
   })
 }
 
+export const initReadOnlySafeSDK = async (chainId: string, safeAddress: string, safeVersion: string): Promise<Safe> => {
```

Comment:
`SafeInfo['version']` can be `null`.

Reply:
We should also initialise this "readonly" SDK as we do with the standard one so that it supports unsupported contracts.